codebase/codebase.go

```diff
@@ -253,3 +278,18 @@ func (m *GormCodebaseRepository) Load(ctx context.Context, id uuid.UUID) (*Codeb
 	}
 	return &obj, nil
 }
+
+// LoadByRepo returns a single codebase found for input repository url
+func (m *GormCodebaseRepository) LoadByRepo(ctx context.Context, repository string) (*Codebase, error) {
```

Comment:
correct @aslakknutsen thanks, 
done in https://github.com/almighty/almighty-core/pull/1241/commits/186e12cabaf69f3eb17de5fff11e621bb92c6e78
Also I have added tests for the same.

Do you think I should index `url` in the relation ?